controlplane/kubeadm/main.go

```diff
@@ -134,6 +135,9 @@ func InitFlags(fs *pflag.FlagSet) {
 	fs.IntVar(&kubeadmControlPlaneConcurrency, "kubeadmcontrolplane-concurrency", 10,
 		"Number of kubeadm control planes to process simultaneously")
 
+	fs.IntVar(&clusterCacheTrackerConcurrency, "clustercachetracker-concurrency", 10,
```

Comment:
clustercachetracker is an implementation detail that's now being exposed to users, could we find a name like `child-cluster-concurrency` or `workload-cluster-concurrency`?

Reply:
I couldn't think of a better name that expresses what we do :). So I thought it's a good idea to use something that express ~ the name of the controller. 

We can use something like workload-cluster-concurrency but it seems misleading to be honest. It's not actually the concurrency of how many workload clusters we can reconcile, right? Also all our clusters are workload clusters.

Is something like clustercache-concurrency better? It doesn't point directly to CCT but it still expresses somewhat that it's the concurrency of the cache we use for clusters.

Reply:
(cc @chrischdi @killianmuldoon we should hold the CAPV v1.8.0 release until we settled on a name :/. I don't want to end up with diverging names and it's not ideal if we have to rename the flag in CAPV after the minor release)

Reply:
@vincepri wdyt about dropping the flag and just hard-coding to 10? 10 should be enough anyway so there is no need for a flag.

I just wanted to avoid that the cct is using a lot of workers if someone needs more workers for the regular cluster reconcilers